tools/installer/README.md

```diff
@@ -8,166 +8,170 @@ To evaluate Occlum in non-docker environment, installers are needed. Occlum prov
 - **occlum-toolchains-\<language\>**: toolchain components for specific language
 - **occlum**: complete package to support all Occlum commands. `occlum-toolchains-gcc` is also installed by default. Please install packages of other programming languages based on your need.
 
-## RPM Installer
+## DEB Installer
 
 ### How to Build
 
 Normally, Occlum installers should be provided together with release. However, users can also build them on their own.
 
-To build RPM packages, a docker container with Occlum CentOS image (based on CentOS 8.2) is needed. Execute below commands under the occlum directory:
+To build deb packages, a docker container with Occlum Ubuntu image (based on Ubuntu 20.04) is needed. Execute below commands under the occlum directory:
 ```
-cd tools/installer/rpm
+cd tools/installer/deb
 make
 ```
-and occlum rpm installer can be found under `build/rpms`.
+and occlum deb installer can be found under `build/debs`.
 
 If a user wants to build his application on a platform installed with Occlum installer, toolchain installers are also needed. To build language specific toolchain installer, just run the command:
 ```
-cd tools/installer/rpm
+cd tools/installer/deb
 make <language option>
 ```
-Now, only `musl-gcc` and `golang` options are supported. And the installer can be found under `build/rpms`.
+Now, only `musl-gcc` and `golang` options are supported. And the installer can be found under `build/debs`.
 
 ### How to Use
 
-RPM installer should be found together with Occlum release package at [this page](https://github.com/occlum/occlum/releases).
-To run Occlum on clean Centos 8, please follow below steps:
+DEB installer should be found together with Occlum release package at [this page](https://github.com/occlum/occlum/releases).
+To run Occlum on clean Ubuntu 18.04, please follow below steps:
 
 **Step 1. Install Prerequisites**
 ```
-yum install -y libcurl-devel openssl-devel fuse-devel fuse-libs autoconf automake cmake libtool make yum-utils gdb python2
-ln -s /usr/bin/python2 /usr/local/bin/python
-dnf config-manager --set-enabled PowerTools
-yum install -y ocaml ocaml-ocamlbuild
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends libcurl4-openssl-dev libssl-dev libprotobuf-dev libfuse-dev autoconf automake make cmake libtool gdb python jq ca-certificates gnupg wget vim
 ```
 
 **Step 2. Install Intel® SGX driver and Intel® SGX PSW**
-Please follow [Intel SGX Installation Guide](https://download.01.org/intel-sgx/sgx-linux/2.13/docs/Intel_SGX_Installation_Guide_Linux_2.13_Open_Source.pdf) to install SGX driver and SGX PSW. SGX SDK is not required. Using RPM installer is recommanded.
+Please follow [Intel SGX Installation Guide](https://download.01.org/intel-sgx/sgx-linux/2.13/docs/Intel_SGX_Installation_Guide_Linux_2.13_Open_Source.pdf) to install SGX driver and SGX PSW. SGX SDK is not required. Using PSW installer is recommanded.
```

Comment:
Please update the link to:
https://download.01.org/intel-sgx/sgx-linux/2.15.1/docs/Intel_SGX_SW_Installation_Guide_for_Linux.pdf

The current Occlum is using SGX 2.15.1

Reply:
Done. Please check out #839 for the update. Thanks for your review.